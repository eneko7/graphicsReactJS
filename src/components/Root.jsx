import { hot } from 'react-hot-loader';
import App from './App/App';

export default hot(module)(App);
