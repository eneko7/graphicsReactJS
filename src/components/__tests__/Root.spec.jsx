import React from 'react';
import renderer from 'react-test-renderer';

import Root from '../Root';

describe('Root', () => {
  it('Root renders correctly', () => {
    const renderedRoot = renderer.create(<Root />).toJSON();
    expect(renderedRoot).toMatchSnapshot();
  });
});
