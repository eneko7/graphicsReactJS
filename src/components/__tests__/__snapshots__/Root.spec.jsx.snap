// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Root renders correctly 1`] = `
<div
  className="main_container"
>
  <footer
    className="footer"
  >
    <span
      className="footer_films"
    >
      FILMS
    </span>
    <span
      className="footer_copyright"
    >
      Copyright Â©
      2018
      FILMS.
      <span
        className="copyright_signature"
      >
        IHAR KARPUK
      </span>
    </span>
  </footer>
</div>
`;
